annotations/license.go

```diff
@@ -0,0 +1,20 @@
+package annotations
+
+import (
+	"time"
+	_ "unsafe"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+)
+
+//go:linkname licenseKey github.com/unidoc/unipdf/v3/common/license.licenseKey
+var licenseKey *license.LicenseKey
```

Comment:
Oh, so that's how you are supposed to use the community license :)

Great to know!